plugins/networkdelay/plugin.go

```diff
@@ -39,6 +41,9 @@ var (
 	myID            string
 	myPublicKey     ed25519.PublicKey
 	originPublicKey ed25519.PublicKey
+
+	// clockEnabled defines if the clock plugin is enabled.
+	clockEnabled bool
```

Comment:
I don't think we need this. We should always use `clock.SyncedTime()` and if the clock is not actually synced (i.e. the plugin is disabled) we just return `time.Now().Add(offset)` where `offset=0`, effectively resulting in `time.Now()`. Makes the code much simpler :)

Reply:
But I would still keep that variable so that we can use it in the networkDelay struct to tell whether the node is using the clock or not. wdyt?

Reply:
Yep that makes sense!